desktop-app/app/components/UserPreferences/index.js

```diff
@@ -128,6 +129,33 @@ export default function UserPreference({
             }
           />
         </div>
+        <div>
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={
+                  userPreferences.screenshotMechanism ===
+                  SCREENSHOT_MECHANISM.V2
+                }
+                onChange={e =>
+                  onChange(
+                    'screenshotMechanism',
+                    e.target.checked
+                      ? SCREENSHOT_MECHANISM.V2
+                      : SCREENSHOT_MECHANISM.V1
+                  )
+                }
+                name="Use improved screenshot mechanism"
+                color="primary"
+              />
+            }
+            label={
+              <span className={cx(styles.preferenceName)}>
+                Use improved screenshot mechanism
```

Comment:
Maybe change this to `Use improved mechanism`? "screenshot" is implicit and it took just one line un the drawer
![image](https://user-images.githubusercontent.com/13673443/91553510-d2f08680-e92d-11ea-86f5-318631423b90.png)
vs
![image](https://user-images.githubusercontent.com/13673443/91553530-db48c180-e92d-11ea-977c-226fa3162131.png)

Reply:
Yeah, this is much better, I'll change it. 👍